Sprintfinity3902/Sprites/Link/LinkDownSprite.cs

```diff
@@ -28,6 +29,9 @@ public LinkDownSprite(Texture2D texture)
             Animation.AddFrame(Sprite1, 1 / 5f);
         }
 
-
+        public override void Draw(SpriteBatch spriteBatch, Vector2 position, Color color, float order)
+        {
+            base.Draw(spriteBatch, position, color, 0.5f);
```

Comment:
Getting rid of this Magic Number might help me understand "order" a little more... I am assuming it is determining the order of what is drawn, but it doesn't seem like everything takes advantage of it. Why does this sprite specifically, and how does the "0.5f" value change it?

Reply:
Thanks bailey... after looking at this I've determined we should not use layer depth because we SpriteSortMode.Deffered which ignores depth. Removing this code from commit.